enclave/enclave-api/src/test/java/com/quorum/tessera/enclave/PayloadEncoderTest.java

```diff
@@ -2078,149 +2086,4 @@ public void withRecipientForEncodedPayloadThatHasEmptyArrayListRecipientKeys() {
 
         assertThat(payload.getRecipientKeys()).containsExactly(PublicKey.from("someKey".getBytes()));
     }
-
-    /**
-     * Do NOT change as this is a copy of the legacy payload encoder. The tests will use these logic to ensure the
-     * legacy encoder is still able to understand new encoded payload and vice versa
-     */
```

Comment:
is this no longer required?

Reply:
It was moved to a separate class https://github.com/ConsenSys/tessera/blob/fbd2cfaeecea4ba50b4d7e477f503d64b4309a67/enclave/enclave-api/src/test/java/com/quorum/tessera/enclave/encoder/LegacyPayloadEncoder.java